androidApp/src/main/java/kmm/rickandmorty/app/android/presentation/components/characters/CharactersScreen.kt

```diff
@@ -49,43 +46,55 @@ fun CharactersScreen() {
         when (charactersState) {
             is Success -> {
                 LazyVerticalGrid(
-                    columns = Adaptive(160.dp),
-                    modifier = Modifier.padding(bottom = 60.dp),
-                    contentPadding = PaddingValues(vertical = 10.dp)
+                    columns = Adaptive(
+                        dimensionResource(id = R.dimen.character_list_item_height)
+                    ),
+                    modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.character_list_bottom_padding)),
+                    contentPadding = PaddingValues(vertical = dimensionResource(id = R.dimen.character_list_items_content_padding))
                 ) {
                     items((charactersState as Success).data) { item ->
                         Character(character = item)
                     }
                 }
             }
-            is Error -> {}
-            Loading -> {}
+            is Error -> {
```

Comment:
Handle Error Case too

Reply:
Will set up error components in a later pull request.